example/feature_demo/buf.gen.yaml

```diff
@@ -1,13 +1,13 @@
 version: v1
 plugins:
-  - remote: buf.build/library/plugins/go:v1.27.1-1
+  - plugin: buf.build/protocolbuffers/go:v1.30.0
```

Comment:
Newer versions of the `buf` cli will use the `plugin` key for both remote and local plugin references.

This also updates the plugins to reference the new Buf-managed plugins, available at:

- https://buf.build/protocolbuffers/go
- https://buf.build/grpc/go